project/plugins.sbt

```diff
@@ -1,18 +1,18 @@
 resolvers += Resolver.url("hmrc-sbt-plugin-releases", url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))(Resolver.ivyStylePatterns)
 
-resolvers += "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
+resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
 
 resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.9.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.10.0")
 
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2")
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
```

Comment:
Not sure if this should be a dependency inside the project. Didn't know that we had this in the project.

This is a kind of dependency that should probably leave in the global configuration for everyone, not inside the project

Reply:
Did not think about it, just updated all plugins to the newest version. I'll have a look if it can be removed.

Reply:
Yeah, I know, that's the reason I approved this :)

This is just to generate the dependency graphs. Codebase is not using it on 100%